src/handlers/comment/handlers/assign.ts

```diff
@@ -13,17 +13,23 @@ export const assign = async (body: string) => {
   const config = getBotConfig();
 
   const payload = _payload as Payload;
+  const organization = payload.organization;
+
   logger.info(`Received '/assign' command from user: ${payload.sender.login}, body: ${body}`);
   const issue = (_payload as Payload).issue;
   if (!issue) {
     logger.info(`Skipping '/assign' because of no issue instance`);
     return "Skipping '/assign' because of no issue instance";
   }
 
+  if (!organization?.id) {
```

Comment:
@pavlovcik This PR basically disables the bot for none-org repos, is it ok?

Reply:
Yea, i thought about that.. And looking at the marketplace, it only allowed me install it to my organization.

So I guess it works unless @pavlovcik states otherwise

Reply:
Before merging the PR. lets clarify about the installation target. 
The ubiquibot is allowed to install to org repos and individual repos so I suggest to use default value 1 if organization id is null.